crates/core/src/catalog.rs

```diff
@@ -117,7 +120,7 @@ pub struct CatalogAddon {
     #[serde(deserialize_with = "null_to_default::deserialize")]
     #[deprecated(since = "0.4.4", note = "Please use game_versions instead")]
     pub flavors: Vec<Flavor>,
```

Comment:
Should we remove `flavors` all together now? It's been deprecated for a long time.